ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/disconnected/private/cco-manual-workload-identity/cucushift-installer-rehearse-azure-ipi-disconnected-private-cco-manual-workload-identity-workflow.metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+	"path": "cucushift/installer/rehearse/azure/ipi/disconnected/private/cco-manual-workload-identity/cucushift-installer-rehearse-azure-ipi-disconnected-private-cco-manual-workload-identity-workflow.yaml",
```

Comment:
I suggest to use `disconnected` + `fullyprivate` to replace `disconnected` + `private`

Reply:
@huangmingxia I think you should reference cucushift-installer-rehearse-vsphere-ipi-disconnected-provision-chain.yaml to opt in the oidc part to complete the install workflow.